src/Entity/Transformer.php

```diff
@@ -177,11 +179,17 @@ public function loadBlock(array $content, PageInterface $page)
 
         $block->setPage($page);
         $block->setId($content['id']);
-        $block->setName($content['name']);
+        if (isset($content['name'])) {
+            $block->setName($content['name']);
```

Comment:
On 4.x we are doing different: https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Entity/Transformer.php#L167